src/signals/incident-management/components/FilterForm/components/CategoryGroups.tsx

```diff
@@ -1,27 +1,45 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2020 - 2022 Gemeente Amsterdam
+// Copyright (C) 2020 - 2023 Gemeente Amsterdam
 import type { FC } from 'react'
 
+import { Accordion } from 'components/Accordion'
 import CheckboxList from 'components/CheckboxList'
 import type { CheckboxListProps } from 'components/CheckboxList'
-import Label from 'components/Label'
 import type {
   ExtendedCategory,
   StructuredCategories,
 } from 'models/categories/selectors'
 import type SubCategory from 'types/api/sub-category'
 
+import type { FilterState } from '../reducer'
+
 type CategoryGroupsProps = Partial<CheckboxListProps> & {
   categories: StructuredCategories
   filterSlugs?: Array<ExtendedCategory | SubCategory>
+  state: FilterState
+}
+
+const getCount = (state: FilterState, key: string, slug: string) => {
```

Comment:
what's the return value here? I should say a number. Then
```
  const mainCategoryCount = mainCategory?.sub?.length ?? ''
```

is not correct

Reply:
either string or number, since we don't want to show anything when it's zero.